app/faucet/views.py

```diff
@@ -24,98 +43,63 @@ def faucet(request):
 
 
 def check_github(profile):
-    if settings.ENV == 'local':
-        creds = False
-    else:
-        creds = True
     user = search_github(profile + ' in:login type:user')
-    if len(user['items']) == 0 or user['items'][0]['login'].lower() != profile.lower():
-       response = {
-        'status': 200,
-        'user': False
-      }
-    else:
-       response = {
-        'status': 200,
-        'user': user['items'][0]
-      }
+    response = {'status': 200, 'user': False}
+    if not len(user['items']) == 0 or not user['items'][0]['login'].lower() != profile.lower():
```

Comment:
this looks different than what was being done before?

Reply:
It is.  I'm checking if not for both cases, defaulting to `{'status': 200, 'user': False}` and only updating `response['user']` if the case is correct ^  - It's performing the same logic, just updating the one value if necessary to be more DRY.